CHANGELOG.md

```diff
@@ -2,6 +2,87 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 49 end 2016-11-14
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+49+Ending+Nov+14%2C+2016%22+label%3Aenhancement)
+
+- Automate
+  - Service Model
+    - Expose ems_events to Vm service model
+    - Expose a group's filters.
+    - Expose authentication_key in EMS service model
+  - Provisioning
+    - Set zone when deliver a service template provision task
+    - Route VM reconfigure request to appropriate region
+  - Services
+    - Expose service power state
+- Platform
+  - High Availabilty: Raise event when failover successful
+  - Chargeback: Add daily to chargeback rate for 'per time' types
+  - Replication: Add logging when the replication set is altered
+  - Logging: Add configurable number of saved logfile rotations
+- Providers
+  - Containers UI: Label based Auto-Tagging UI
+  - Middleware (Hawkular)
+    - Add support to overwrite an existing deployment
+  - Networks
+    - Nuage: UI for Network elements
+    - Add Network Topology button for the Load Balancer class
+  - Red Hat Enterprise Virtualization Manager
+      - Enable VM reconfigure disks for supported rhevm version
+      - Migrate support
+  - VMware vCloud UI: Event monitoring
+- REST API
+  - Add IDs to Dialog Content
+  - Actions support
+  - Conditions support
+  - Support for /api/requests approve and deny actions
+  - Service Dialogs Copy API
+  - MiqPolicies support
+  - Service Request Delete
+- SmartState: Support NFS41 datastore
```

Comment:
To differentiate this from "datastore" smartstate, Id change this to something like:
SmartState: Support analysis of VMs residing on NFS41 datastores.